docs/modules/ROOT/pages/cli/file-based-config.adoc

```diff
@@ -0,0 +1,5 @@
+= File Based Configuration
+
+File-based configuration is used to set command flags. Flag values do not need to be entered on a regular basis. The file is read on Kamel startup and the flags are set accordingly.
+
+To configure this flag, create a file named kamel-config.yaml on the same directory as your integration. The file must contain a yaml structure as shown in ./kamel-config.yaml.
```

Comment:
You should remove the empty `kamel-config.yaml` file. Rather the doc should include the content of a config file and some examples on how to use the cli with the config file.

You can look at other camel-k doc such as:
https://camel.apache.org/camel-k/next/cli/modeline.html
to have an idea on how it should be written.

Reply:
Also, you should write that `kamel-config.yaml` is the default file name and you can change it with the `KAMEL_CONFIG_NAME` env var.

You should also explain here that the default config paths are: `.`, `./.kamel`, and `$HOME/.kamel/` and you can change them with the `KAMEL_CONFIG_PATH` env var as well.

Reply:
Thank you.